src/diffusers/models/embeddings.py

```diff
@@ -200,7 +199,7 @@ def __init__(
         sample_proj_bias=True,
     ):
         super().__init__()
-        linear_cls = nn.Linear if USE_PEFT_BACKEND else LoRACompatibleLinear
+        linear_cls = nn.Linear
```

Comment:
in the future, we could simply declare `nn.Linear(xxx)` instead of having `xxx_cls` all over the places but I really think it's not urgent

Reply:
@sayakpaul we can ask community to do this if you want:)